README.md

```diff
@@ -77,28 +77,34 @@ export default () => (
 | innerBorderWidth               | `number`           | 0                         | Width of the roulette's inner border line (0 represents no inner border line)                         |
 | radiusLineColor                | `string`           | 'black'                   | Color of the radial lines that separate each option                                                   |
 | radiusLineWidth                | `number`           | 5                         | Width of the radial lines that separate each option (0 represents no radial lines)                    |
-| fontSize                       | `number`           | 20                        | Font size of the option string                                                                        |
+| fontFamily                     | `string`           | 'Helvetica, sans-serif '  | Global font family of the option string                                                               |
```

Comment:
Add some details about the font format. Nothing too long, just for the user to know which strings are available. You could also provide a link to some google documentation webpage, like "these are the available fonts: ..."